### PR TITLE
Update Windows Virtual Desktop gallery entry point

### PR DESCRIPTION
**Pull request by xiangtin**
## Summary
1. Rename the "Windows Virtual Desktop Insights" to "Azure Virtual Desktop Insights".
2. Redirect the "Azure Virtual Desktop Insights" to "Workbooks/Windows Virtual Desktop/AtScale/WVD Insights" workbook.

## Validation
https://ms.portal.azure.com/?feature.workbookGalleryBranch=xixiao%2Fgalleryupdate#view/Microsoft_Azure_Monitoring/AzureMonitoringBrowseBlade/~/workbooks

image

### CODE REVIEW

File: gallery/workbook/Azure Monitor.json [hunk 1]
```diff
@@ -212,10 +212,10 @@
 			"name": "Insights",
 			"templates": [
 				{
-					"id": "Workbooks/Windows Virtual Desktop/WVD Insights",
+					"id": "Workbooks/Windows Virtual Desktop/AtScale/Overview",
 					"author": "Microsoft",
 					"description": "Diagnostics, performance, and usage for Windows Virtual Desktop",
```
Comment (mike-mo) [inline gallery/workbook/Azure Monitor.json:217]: Azure instead of Windows on this line too

Comment (xiangtin) [reply gallery/workbook/Azure Monitor.json]: Updated the description for the gallery.

**Review by gardnerjr — APPROVED** [review]
approved for azure monitor.json

File: gallery/workbook/Azure Monitor.json (hunk 1 above)
Comment (mike-mo) [reply gallery/workbook/Azure Monitor.json]: Looks like line 217 could still use an update though

**Review by mike-mo — APPROVED** [review]

**Pull request merged**